src/geometry/euclidean/oriented_angle.lean

```diff
@@ -43,56 +43,55 @@ noncomputable theory
 open_locale real
 open_locale real_inner_product_space
 
-namespace orthonormal
+namespace orthonormal_basis
 
 variables {V : Type*} [inner_product_space ℝ V]
-variables {b : basis (fin 2) ℝ V} (hb : orthonormal ℝ b)
-include hb
+variables (b : orthonormal_basis (fin 2) ℝ V)
```

Comment:
This change looks to create a lot of noise in the diff; is it straightforward to split to a separate PR?

Reply:
Yes, see the prereq PR #16475.